converter/stitching/stitcher.go

```diff
@@ -126,9 +126,9 @@ func (s *stitcher) stitchFlow(flow input.Flow) error {
 		if s.shouldMerge(&newSessAgg, &oldSessAgg) {
 			matchFound = true
 
-			f_FlowStartMilliseconds := oldSessAgg.FlowStartMilliseconds()
-			nextMatch_FlowStartMilliseconds := oldSessAgg.FlowStartMilliseconds()
-			f_FlowEndMilliseconds := oldSessAgg.FlowEndMilliseconds()
+			f_FlowStartMilliseconds := newSessAgg.FlowStartMilliseconds()
+			nextMatch_FlowStartMilliseconds := newSessAgg.FlowStartMilliseconds()
```

Comment:
The f_ variables should be based on newSessAgg, the nextMatch_  variales should be based on oldSessAgg. 

We are matching a new session aggregate against ones we've seen in the past.